errors/list_test.go

```diff
@@ -145,6 +145,34 @@ func TestErrorListFlattensOtherErrorList(t *testing.T) {
 	}
 }
 
+func TestErrorListsWithMatchingErrors(t *testing.T) {
+	const (
+		kind1 errors.Kind = "kind1"
+		kind2 errors.Kind = "kind2"
+		kind3 errors.Kind = "kind3"
+	)
+
+	t.Run("empty list match", func(t *testing.T) {
+		assert.IsTrue(t, errors.Is(errors.L(), errors.L()))
+	})
+
+	errs1 := errors.L(errors.E(kind1), errors.E(kind2), errors.E(kind3))
+
+	t.Run("exact errors match", func(t *testing.T) {
+		errs2 := errors.L(errors.E(kind1), errors.E(kind2), errors.E(kind3))
```

Comment:
Now when we want to check for error lists it should be easier

Reply:
yeah much better, I know the pain, I should have done that in my previous PRs...